tests/seek.txt

```diff
@@ -0,0 +1 @@
+aaccccAAbbbbbbbb
```

Comment:
It is probably not necessary to include this file in the commit, because the example will create the file.

* I suggest adding `tests/seek.txt` to `.gitignore`.
* If you would like to keep a copy of the data in the tree for comparison purposes, consider naming it something else, such as `tests/seek_output.txt`, then you can `diff tests/seek.txt tests/seek_output.txt` to make sure the output is correct.

Reply:
My initial intention was to commit the file to give users reading the example the output to verify that it works. Adding it as a test for travis already ensures that, so I removed the file.